s3router.js

```diff
@@ -25,9 +25,9 @@ function S3Router(options) {
 
     /**
      * Redirects image requests with a temporary signed URL, giving access
-     * to GET an image.
+     * to GET an upload.
      */
-    router.get(/\/img\/(.*)/, function(req, res) {
+    router.get(/\/upload\/(.*)/, function(req, res) {
```

Comment:
I like the other `router.get` as well - easier to read and clean up later.

Use `/s3/uploads` for the public route.  The `/s3/img` is just for backward compatibility.